core-java-modules/core-java-io-conversions-2/src/test/java/com/baeldung/filetofrombytearray/FileToByteArrayUnitTest.java

```diff
@@ -0,0 +1,63 @@
+package com.baeldung.filetofrombytearray;
+
+import static org.junit.Assert.assertArrayEquals;
```

Comment:
Please use the method from JUnit 5 instead of the one from JUnit 4.

Reply:
Done